tests/loggers/test_mlflow_logger.py

```diff
@@ -640,5 +636,49 @@ def test_mlflow_ignore_metrics(tmp_path, device):
     metric_file = run_file_path / Path('metrics') / Path('system/cpu_utilization_percentage')
     assert not os.path.exists(metric_file)
 
-    # Undo the setup to avoid affecting other test cases.
-    mlflow.set_system_metrics_sampling_interval(None)
+
+@device('cpu')
+def test_mlflow_ignore_hyperparameters(tmp_path, device):
+    mlflow_uri = tmp_path / Path('my-test-mlflow-uri')
+    experiment_name = 'mlflow_logging_test'
+    test_mlflow_logger = MLFlowLogger(tracking_uri=mlflow_uri,
+                                      experiment_name=experiment_name,
+                                      log_system_metrics=False,
+                                      ignore_hyperparameters=['num*', 'mlflow_run_id', 'nothing'])
+
+    dataset_size = 64
+    batch_size = 4
+    num_batches = 4
+    eval_interval = '1ba'
+
+    Trainer(model=SimpleConvModel(),
+            loggers=test_mlflow_logger,
+            train_dataloader=DataLoader(RandomImageDataset(size=dataset_size), batch_size),
```

Comment:
you can just make the dataloaders empty lists too, since you're not using them for anything.